package.json

```
@@ -20,9 +20,10 @@
   "scripts": {
     "build": "node bin/octokit-webhooks.js check --cached",
     "pretest": "npm run -s lint",
-    "test": "npm run build && node test.js",
-    "lint": "prettier --check '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' *.md package.json index.json",
-    "lint:fix": "prettier --write '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' *.md package.json index.json"
+    "test": "npm run build && npm run validate && node test.js",
+    "lint": "prettier --check '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' 'payload-schemas/schemas/*.json' 'payload-schemas/schemas/common/*.json' 'payload-schemas/*.js' *.md package.json index.json",
+    "lint:fix": "prettier --write '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' 'payload-schemas/schemas/*.json' 'payload-schemas/schemas/common/*.json' 'payload-schemas/*.js' *.md package.json index.json",
```

`'payload-examples/**/*.json'` should find all JSON files in all folders within `payload-examples`

```suggestion
    "lint": "prettier --check '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' 'payload-schemas/*.js' *.md package.json index.json",
    "lint:fix": "prettier --write '{bin,lib}/*.js' '*.{js,md}' 'payload-examples/**/*.json' 'payload-schemas/*.js' *.md package.json index.json",
```